targets/TARGET_STM/TARGET_STM32F4/device/stm32f4xx_hal_dma.c

```diff
@@ -761,6 +761,9 @@ void HAL_DMA_IRQHandler(DMA_HandleTypeDef *hdma)
       
       /* Update error code */
       hdma->ErrorCode |= HAL_DMA_ERROR_TE;
+
+      /* Change the DMA state */
+      hdma->State = HAL_DMA_STATE_ERROR; // FIX
```

Comment:
Yes I know... I have added this in order to see where we have modified the original ST HAL file...